datafusion/sqllogictest/test_files/information_schema.slt

```diff
@@ -296,6 +297,7 @@ datafusion.execution.parquet.bloom_filter_fpp NULL (writing) Sets bloom filter f
 datafusion.execution.parquet.bloom_filter_ndv NULL (writing) Sets bloom filter number of distinct values. If NULL, uses default parquet writer setting
 datafusion.execution.parquet.bloom_filter_on_read true (writing) Use any available bloom filters when reading parquet files
 datafusion.execution.parquet.bloom_filter_on_write false (writing) Write bloom filters for all columns when creating parquet files
+datafusion.execution.parquet.coerce_int96 NULL (reading) If true, parquet reader will read columns of physical type int96 as originating from a different resolution than nanosecond. This is useful for systems like Spark which stores its 64-bit timestamps as microsecond resolution, so it can write values with a larger date range than 64-bit timestamps with nanosecond resolution.
```

Comment:
nit: This was sort of confusing for me. Don't know if my suggestion is clearer though.